tests/file_storage/tests.py

```diff
@@ -911,6 +911,9 @@ def test_callable_storage_file_field_in_model(self):
         self.assertEqual(obj.storage_callable.storage, temp_storage)
         self.assertEqual(obj.storage_callable.storage.location, temp_storage_location)
         self.assertIsInstance(obj.storage_callable_class.storage, BaseStorage)
+        # Deconstructing a FileField with a callable Storage should not evaluate the callable
+        name, path, args, kwargs = obj._meta.get_field('storage_callable').deconstruct()
+        self.assertNotIsInstance(kwargs['storage'], BaseStorage)
```

Comment:
This is probably worth it's own test-case. 

(For ref: there are similar tests on ln70)